examples/nl-normal-address.html

```diff
@@ -0,0 +1,132 @@
+ *
+ */
+?>
+
+<script src="https://code.jquery.com/jquery-1.11.3.js"></script>
+<link rel="stylesheet" href="../css/myparcel.css">
+
+<div id="mypa-spinner"></div>
+<div id="mypa-message"></div>
+<div id="mypa-location-details"></div>
+
+<form id="mypa-delivery-option-form" class="" action="../" enctype="multipart/form-data" method="post" target="_self">
+	<div class="mypa-fieldset">
+		<div id="mypa-send-per-carrier">
+			<div id="mypa-txt-send-per-carrier">Bezorgopties</div>
+		</div>
+		<ul id="mypa-pickup-selector">
+			<li>
+				<input name="mypa-deliver-or-pickup" id="mypa-deliver-pickup-deliver" value="mypa-deliver" type="radio">
+				<label for="mypa-delivery-date">Bezorgen op</label>
+			</li>
+		</ul>
+		<ul id="mypa-delivery-selectors-nl">
+			<li>
+				<div class="full-width" id="mypa-delivery-date">
+					<select name="mypa-delivery-date" id="mypa-select-date">
+						<option value="">Geen datum</option>
+					</select>
+				</div>
+			</li>
+			<li class="mypa-delivery-option method-myparcel-delivery-morning-div">
+				<input name="shipping-method" id="method-myparcel-delivery-morning" type="radio" value="myparcel-morning">
+				<label for="method-myparcel-delivery-morning">Ochtendlevering <span id="mypa-morning-delivery"></span></label>
+			</li>
+			<li id="mypa-postnl-delivery" class="mypa-delivery-option method-myparcel-flatrate-div">
+				<input name="shipping-method" id="method-myparcel-flatrate" type="radio" value="myparcel-flatrate">
+				<label for="method-myparcel-flatrate">Thuislevering</label>
+			</li>
+
+			<li class="mypa-delivery-option method-myparcel-delivery-evening-div">
+				<input name="shipping-method" id="method-myparcel-delivery-evening" type="radio" value="myparcel-delivery-evening" >
+				<label for="method-myparcel-delivery-evening">Avondlevering <span id="mypa-evening-delivery"></span></label>
+			</li>
+		</ul>
+		<ul id="mypa-extra-delivery-options">
+			<li id="mypa-postnl-signature">
+				<input name="myparcel-signature-selector" id="mypa-signature-selector" type="checkbox" value="myparcel-signature-selector" >
+				<label for="mypa-signature-selector">Handtekening<span id="mypa-signature-price"></span></label>
+			</li>
+			<li id="mypa-postnl-only-recipient">
+				<input name="method-myparcel-only-recipient-selector" id="mypa-only-recipient-selector" type="checkbox" value="myparcel-recipient-only-selector" >
+				<label for="mypa-only-recipient-selector">Alleen geadresseerde<span id="mypa-recipient-only"></span></label>
+			</li>
+		</ul>
+		<ul id="mypa-pickup-location-selector">
+			<li>
+				<input name="mypa-deliver-or-pickup" id="mypa-deliver-pickup-pickup" value="mypa-pickup" type="radio">
+				<label for="mypa-pickup-location">Afhalen op locatie<span id="mypa-price-pickup"></span></label>
+			</li>
+			<li>
+				<div class="full-width" id="mypa-pickup-options">
+					<select name="mypa-delivery-date" id="mypa-pickup-location">
+						<option value="">Geen Locatie</option>
```

Comment:
Stel iemand kan hier kiezen voor `Geen locatie`, wat gebeurd er dan?

Reply:
De optie `geen locatie` is niet zichtbaar, dit wordt overschreven door javascript. De bedoeling is dat er `geen locatie` te zien is als er geen verbinding gemaak kan worden met api.myparcel.nl.